### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by alireza1385mamani** [Windscribe/Desktop-App#50]
Hello and thank you for building this great VPN and app but there are accessibility issues with nvda screen reader on Windows with this app we cannot access locations or maybe by even connecting and disconnecting the VPN NZ is not working because nvda 10 not read anything at all all for us can you please make it accessible with nvda screen reader thank you

**Comment by bernerdad** [comment]
Hi.  This improvement is on our board for the 2.7 release, which should be available in the last quarter of 2022.

**Comment by alireza1385mamani** [comment]
OK. thanks.
